iota/crypto/types.py

```diff
@@ -75,6 +77,13 @@ class Seed(TryteString):
   References:
     - https://forum.iota.org/t/why-arent-seeds-longer-than-81-trytes-more-secure/1278
   """
+  def __init__(self, trytes=None):
+    # type: (Optional[TrytesCompatible]) -> None
+    if trytes:
+        if len(trytes) > 81:
+         warnings.warn("Beyond 81 trytes, you can't increase the security of your seed, see - https://forum.iota.org/t/why-arent-seeds-longer-than-81-trytes-more-secure/1278")
```

Comment:
Please make the indenting consistent — 2 spaces per level.

Also, the PyOTA codebase generally conforms to [PEP-8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length), in this case, max line length s/b 79 chars.

These changes make the code formatting more consistent, and the overall width of the code is narrower, which reduces [cognitive load](https://en.wikipedia.org/wiki/Cognitive_load).